checkUserInput.php

```diff
@@ -1,11 +1,14 @@
 <?php
 require_once "functions/allFunctions.php";
 
+if (!isset($_POST['species'])){
+    header("location: index.php");
```

Comment:
You should call exit() after using a header like this. I also believe that the l in location should be L